src/main/webapp/favicon.svg

```diff
@@ -0,0 +1,25 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.com/svgjs" width="67.942429mm" height="67.942429mm"><svg xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:cc="http://creativecommons.org/ns#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd" xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape" width="67.942429mm" height="67.942429mm" viewBox="0 0 67.942429 67.942429" version="1.1" id="SvgjsSvg1010" inkscape:version="1.0.1 (3bc2e813f5, 2020-09-07)">
+  <title id="SvgjsTitle1009">testKent</title>
+  <defs id="SvgjsDefs1008"></defs>
+  <sodipodi:namedview id="SvgjsSodipodi:namedview1007" pagecolor="#ffffff" bordercolor="#666666" borderopacity="1.0" inkscape:pageopacity="0.0" inkscape:pageshadow="2" inkscape:zoom="1.414489" inkscape:cx="-1.1056881" inkscape:cy="139.88778" inkscape:document-units="mm" inkscape:current-layer="g3365" inkscape:document-rotation="0" showgrid="false" fit-margin-top="0" fit-margin-left="0" fit-margin-right="0" fit-margin-bottom="0" inkscape:window-width="1920" inkscape:window-height="1017" inkscape:window-x="1912" inkscape:window-y="-8" inkscape:window-maximized="1"></sodipodi:namedview>
+  <metadata id="SvgjsMetadata1006">
+    <rdf:rdf>
+      <cc:work rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type rdf:resource="http://purl.org/dc/dcmitype/StillImage"></dc:type>
+        <dc:title>testKent</dc:title>
+      </cc:work>
+    </rdf:rdf>
+  </metadata>
+  <g inkscape:label="Layer 1" inkscape:groupmode="layer" id="SvgjsG1005" transform="translate(-42.062714,-127.96269)">
```

Comment:
please remove all inkscape tags, they seem unnecessary

Reply:
Is Done.